installer/datafiles/base_omsagent.data

```diff
@@ -18,15 +18,16 @@ MAINTAINER:              'Microsoft Corporation'
 %Files
 /etc/logrotate.d/omsagent;                                              installer/conf/logrotate.conf;                         644; root; root
 
-/etc/opt/microsoft/omsagent/sysconf/README;                             installer/conf/sysconf.readme;                         644; root; root
-/etc/opt/microsoft/omsagent/sysconf/omsagent.conf;                      installer/conf/omsagent.conf;                          644; root; root
-/etc/opt/microsoft/omsagent/sysconf/installinfo.txt;                    installer/conf/installinfo.txt;                        644; root; root; conffile
-/etc/opt/microsoft/omsagent/sysconf/omi_mapping.json;                   installer/conf/omi_mapping.json;                       644; root; root
-/etc/opt/microsoft/omsagent/sysconf/omsagent.d/change_tracking.conf;    installer/conf/omsagent.d/change_tracking.conf;        644; root; root
-/etc/opt/microsoft/omsagent/sysconf/omsagent.d/monitor.conf;            installer/conf/omsagent.d/monitor.conf;                644; root; root
-
-/etc/opt/microsoft/omsagent/sysconf/omsagent.d/oms.conf;                installer/conf/oms.conf;                               644; root; root
-/etc/opt/microsoft/omsagent/sysconf/omsagent.d/collectd.conf;           installer/conf/omsagent.d/collectd.conf;               644; root; root
+/etc/opt/microsoft/omsagent/sysconf/README;                                     installer/conf/sysconf.readme;                                 644; root; root
```

Comment:
Can you follow indentation on the rest of the file, except for the lines that are too long?